tests/statistics.cc

```diff
@@ -245,7 +246,7 @@ BOOST_AUTO_TEST_CASE(StatisticsTestAllNanButOne) { /* parasoft-suppress  LsstDm-
 
 }
 
-BOOST_AUTO_TEST_CASE(StatisticsTestImages) { /* parasoft-suppress  LsstDm-3-2a LsstDm-3-4a LsstDm-4-6 LsstDm-5-25 "Boost non-Std" */
+BOOST_AUTO_TEST_CASE(StatisticsTestImages, * utf::disabled()) { /* parasoft-suppress  LsstDm-3-2a LsstDm-3-4a LsstDm-4-6 LsstDm-5-25 "Boost non-Std" */
```

Comment:
I don't think this is doing what you think then. If I run this test file without {{afwdata}} setup it runs the 3 enabled tests. If I setup {{afwdata}} it still only runs the 3 tests.